charts/langsmith/examples/read_only_config.yaml

```diff
@@ -163,9 +163,9 @@ clickhouse:
   statefulSet:
     podSecurityContext:
       runAsNonRoot: true
-      runAsUser: 1000
-      runAsGroup: 1000
-      fsGroup: 1000
+      runAsUser: 101
```

Comment:
why this change?

Reply:
was incorrect, we use the right value in ci but not in the example